trunk/source/CACSL2BoogieTranslator/src/de/uni_freiburg/informatik/ultimate/cdt/translation/implementation/CLocation.java

```diff
@@ -176,4 +177,12 @@ private IASTNode getMergedNode(final CLocation otherCloc) {
 		}
 	}
 
+	@Override
+	public String getFunction() {
+		final IASTFunctionDefinition scope = CdtASTUtils.findScope(mNode);
+		if (scope == null) {
+			return null;

```

Comment:
What does this case signify? Should we throw an error instead?

Reply:
I guess null is the global scope, and then this cannot be part of a function, right?

Reply:
Exactly, global locations cannot be assigned any function, so I think returning null should be fine here.